cardano_node_tests/tests/tests_conway/test_conway_extras.py

```diff
@@ -0,0 +1,925 @@
+            tx_files=tx_files,
+            deposit=deposit_amt,
+        )
+        stake_addr_info = cluster.g_query.get_stake_addr_info(pool_user.stake.address)
+
+        assert stake_addr_info.vote_delegation == governance_utils.get_drep_cred_name(
+            drep_id=drep1.drep_id
+        ), "Votes are NOT delegated to the correct DRep 1 placed at last of certificates list."
+        reqc.cli087.success()
+
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.parametrize("drep", ("always_abstain", "always_no_confidence", "custom"))
+    @pytest.mark.testnets
+    @pytest.mark.smoke
+    def test_no_delegation_without_stake_registration(
+        self,
+        cluster: clusterlib.ClusterLib,
+        payment_addr: clusterlib.AddressRecord,
+        pool_user: clusterlib.PoolUser,
+        custom_drep: governance_utils.DRepRegistration,
+        drep: str,
+    ):
+        """Test Change delegation to different dreps.
+
+        * Create 2 Dreps
+        * Create vote delegation certifcate for first drep
+        * Submit certificate
+        * check that the delegation is of correct drep id
+        * Change delegation to drep2 and submit certificate
+        * Check vote delegation is updated to second drep
+        """
+        temp_template = common.get_test_id(cluster)
+        deposit_amt = cluster.g_query.get_address_deposit()
+
+        reqc.cli088.start(url=helpers.get_vcs_link())
+        # Create vote delegation cert
+        deleg_cert = cluster.g_stake_address.gen_vote_delegation_cert(
+            addr_name=f"{temp_template}_addr1",
+            stake_vkey_file=pool_user.stake.vkey_file,
+            drep_key_hash=custom_drep.drep_id if drep == "custom" else "",
+            always_abstain=drep == "always_abstain",
+            always_no_confidence=drep == "always_no_confidence",
+        )
+
+        tx_files = clusterlib.TxFiles(
+            certificate_files=[deleg_cert],
+            signing_key_files=[payment_addr.skey_file, pool_user.stake.skey_file],
+        )
+
+        # Make sure we have enough time to finish the delegation in one epoch
+        clusterlib_utils.wait_for_epoch_interval(
+            cluster_obj=cluster, start=1, stop=common.EPOCH_STOP_SEC_LEDGER_STATE
+        )
+
+        # Expecting error as stake address is not registered
+        with pytest.raises(clusterlib.CLIError) as excinfo:
+            clusterlib_utils.build_and_submit_tx(
+                cluster_obj=cluster,
+                name_template=temp_template,
+                src_address=payment_addr.address,
+                use_build_cmd=True,
+                tx_files=tx_files,
+                deposit=deposit_amt,
+            )
+
+        err_msg = str(excinfo.value)
+        assert "StakeKeyNotRegisteredDELEG" in err_msg, err_msg
+        reqc.cli088.success()
+
+    @allure.link(helpers.get_vcs_link())
+    @submit_utils.PARAM_SUBMIT_METHOD
+    @common.PARAM_USE_BUILD_CMD
+    @pytest.mark.dbsync
```

Comment:
dbsync mark should only be used when dbsync is checked

Reply:
Thank you for the feedback. It has been removed in the latest push.